addons/udes_stock/wizard/change_quant_location.py

```diff
@@ -22,11 +21,57 @@ def _default_stock(self):
     )
     reference = fields.Char('Reference')
 
+    @api.multi
+    def check_reserved(self):
+        """Check if we are moving reserved pacakges and if that is permitted"""
+        self.ensure_one()
+        Package = self.env['stock.quant.package']
+
+        packages = Package.browse(self.env.context['active_ids'])
```

Comment:
Nit: Mix of single/double quotes here and in `_check_reserved_packages_permitted`. Although as this is an existing file can understand we might not want to run black.